src/sp/it/pl/layout/widget/EmptyWidget.kt

```diff
@@ -37,11 +37,11 @@ class EmptyWidget: Widget<EmptyWidget>("Empty", emptyWidgetFactory), Controller
 
     override fun refresh() {}
 
-    override fun getWidget() = this
+    override fun getOwnerWidget() = this
```

Comment:
please try to keep refactoring and reformatting separate from functional PRs, since they clutter up the changelist and make reviewing a lot more tedious by obscuring the actual changes.

Reply:
I understand but this was a more or less necessary change. The problem was that for widgets to use configurable properties I wanted the parent Controller - ClassController to be kotlin file. However I had problem where some properties were conflicting with interface definitions (val widget vs getWidget(), etc) and I had to change the interface.

I'll be more careful in the future.

Reply:
Maybe you can try to commit simple refactors (just changing names or refactoring code) directly to master and then simply merge master into the branch you work on.

Reply:
But they are related to the task. I dont know, seems like splitting what should not be split and adding more overhead.

Reply:
I think having this split into commits is an ideal scenario. Branches are way abstract for that.